package.json

```diff
@@ -70,7 +70,7 @@
     "@bbc/psammead-inline-link": "^1.3.11",
     "@bbc/psammead-locales": "^3.0.0",
     "@bbc/psammead-media-indicator": "^2.6.4",
-    "@bbc/psammead-media-player": "^1.1.1-alpha.7",
+    "@bbc/psammead-media-player": "^2.0.0",
```

Comment:
I thought the media player isn't a full release version yet so shouldn't it still be an alpha component?

Reply:
It is a full release now. We've fixed all the accessibility issues